src/reference-manual/types.Rmd

```diff
@@ -1318,9 +1329,49 @@ otherwise expressible as simple expressions (e.g., iterative or
 recursive algorithms).
 
 
-### Scope within expressions {-}
+#### Scope within expressions {-}
 
 Any variable that is in scope and any function that is available in
 the block in which the compound declaration and definition appears may
 be used in the expression on the right-hand side of the compound
 declaration and definition statement.
+
+
+### Declaring multiple variables at once {-}
```

Comment:
Either pull this up or remove the nested sections.  

My inclination would be to put this up with the basic examples after the first batch of simple examples.  It's also OK in its own section like this.